profiles/desktop.py

```diff
@@ -1,6 +1,6 @@
 # A desktop environment selector.
-
 import archinstall
+from archinstall.lib.menu import Menu
```

Comment:
If you can, add `from archinstall.lib.menu import Menu` into [`__init__.py` here](https://github.com/archlinux/archinstall/blob/master/archinstall/__init__.py) instead to expose it in `archinstall.Menu` for convenience when using archinstall in dotfiles.